packages/react-aria-components/docs/GridList.mdx

```diff
@@ -168,6 +169,16 @@ import {GridList, Item, Button} from 'react-aria-components';
       border-start-start-radius: 0;
       border-start-end-radius: 0;
     }
+
+    & .react-aria-Item[aria-selected=true]:has(+ .react-aria-DropIndicator + [aria-selected=true]) {
```

Comment:
fixes the case where contiguous selections were being broken up by hidden drop indicators

Reply:
Would it make sense to combine these rules with the above ones? Also I think the `&` is not required here.

Reply:
yeah, & was copied from above lines